app/views/pages/_editor_guide_text.html.erb

```diff
@@ -35,6 +35,7 @@
     </pre>
     <h3><strong>Inline Images</strong></h3>
     <p>
+      When adding images/gifs to posts and comments, please note that there is a limit of 200 megapixels per frame/page.
```

Comment:
I think this applies only to GIFs, not general images for which the upload size limit applies.

Thanks for the PR!

Reply:
Thank you for the swift response! I've updated the PR